tests/test_sites.py

```diff
@@ -213,10 +218,12 @@ def test_get_student_modules_for_course_in_site(self):
 
         assert get_user_model().objects.count() == 0
         user = UserFactory()
-        UserOrganizationMappingFactory(user=user,
-                                       organization=self.organization)
+        EdlyUserProfileFactory(
+            user=user,
+            edly_sub_organizations=[self.edly_sub_organization]
+        )
 
-        sm_count = 2
+        sm_count = 1
```

Comment:
why this court is reduce to 1?

Reply:
The student module has a `unique_together` of `course_id`, `student_id` and `state`. Here, two student modules were being created for the same user and student. So, I either do this or add a `module_state_key` which is a usage key that would require a module to be created as well. So, for the sake of simplicity and lower amount of changes I did this. Do tell me If I should do it some other way.

Reply:
@rehan99000 how come it was working before?

Reply:
nit: replace `sm` with `student_module` everywhere in this test case.